script.js

```diff
@@ -0,0 +1,47 @@
+const booksList = document.querySelector('#book-list');
+let booklist = [];
+let bookHtml = '';
+if (JSON.parse(localStorage.getItem('books')) == null) {
+  const bookObj = { allbook: booklist };
+  localStorage.setItem('books', JSON.stringify(bookObj));
+} else {
+  booklist = JSON.parse(localStorage.getItem('books')).allbook;
+
+  booklist.forEach((item, index) => {
+    bookHtml += `
+<p>${item.title}</p>
+<p>${item.author}</p>
+<button type="button" class="remove-btn" id="${index}">Remove</button>
+<hr>`;
+  });
+  booksList.innerHTML = bookHtml;
+}
+
+const bookForm = document.querySelector('#book-form');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+
+bookForm.addEventListener('submit', () => {
+  const newBook = {
+    title: bookTitle.value,
+    author: bookAuthor.value,
+  };
+
+  const obj = JSON.parse(localStorage.getItem('books'));
+
+  obj.allbook.push(newBook);
+
+  booklist = obj.allbook;
```

Comment:
- [ ] I have tried adding a book and it wasn't displayed on the UI because you are not saving books to localStorage after saving. Consider adding the line below:

```JavaScript
localStorage.setItem('books', JSON.stringify(obj));
```